src/tools/blast/components/results/BlastResult.tsx

```diff
@@ -112,16 +110,38 @@ const BlastResult = () => {
 
   const data = useMemo(() => enrich(blastData, apiData), [blastData, apiData]);
 
-  if (blastLoading) return <Loader />;
+  if (blastLoading) {
+    return <Loader />;
+  }
+
+  if (blastError || !blastData) {
+    return <ErrorHandler status={blastStatus} />;
+  }
+
+  // Deciding what should be displayed on the sidebar
+  const facetsSidebar = <BlastResultSidebar loading={apiLoading} data={data} />;
+  const emptySidebar = (
+    <div className="sidebar-layout__sidebar-content--empty" />
+  );
+  let sidebar;
+
+  switch (match.params.subPage) {
+    case 'text-output':
```

Comment:
Could define these as variables/symbols to avoid potential typo related issues later